### ISSUE TITLE
Add links to full page versions of documents

### ISSUE THREAD

**Issue by srstsavage** [ioos/sos-guidelines-hugo#8]
Many of the sos-guidelines documents are dense (SOS templates, list of tests, etc) and it's helpful to give them as much screen real estate as possible when viewing them. Currently the published pages at http://ioos.github.io/sos-guidelines have a large left-side pane which takes away a lot of the available viewing room.

Is it possible to provide links to open new tabs for these documents, and to make those new tabs full page and without any layout templates? Markdown should still be rendered.

**Comment by abirger** [comment]
I believe that it is not possible to make various pages look differently with the current web site generator, so it will require a new layout (theme) for the web site ioos.github.io/sos-guidelines that is more suitable for the documents.

**Comment by abirger** [comment]
@shane-axiom , could you please check the new version of the documentation site at http://abirger.github.io/sos-guidelines/ ? I think I did what you wanted with that sliding sidebar...

**Comment by srstsavage** [comment]
Yeah, that looks great. Even if the sidebar started out expanded, the ability to collapse it still allows the user to use all available screen space for large documents.

**Comment by srstsavage** [comment]
This still needs a merge...
